src/msgs/messages2.sql

```diff
@@ -897,6 +897,7 @@ Data source : @4', NULL, NULL)
 ('already_opened', 'lockDatabaseFile', 'unix.cpp', NULL, 0, 787, NULL, 'Database already opened with engine instance, incompatible with current', NULL, NULL);
 ('bad_crypt_key', NULL, 'CryptoManager.cpp', NULL, 0, 788, NULL, 'Invalid crypt key @1', NULL, NULL);
 ('encrypt_error', NULL, 'CryptoManager.cpp', NULL, 0, 789, NULL, 'Page requires encyption but crypt plugin is missing', NULL, NULL);
```

Comment:
There is a (traditional) problem with error codes already used in next version. Codes 790-884 of JRD facility is already used at master branch. We could add code 885 (as in master), creating big gap in numeration for v3, or, use isc_random in v3. Opinions ?